lib/codegen/fromcto/csharp/csharpvisitor.js

```diff
@@ -505,7 +498,7 @@ class CSharpVisitor {
 
         // Apply pascal casing.
         if(parameters?.pascalCase) {
-            name = camelCase(name, { pascalCase: true });
+            name = this.toCase(name, true);
```

Comment:
can we make this a default parameter?

Reply:
The default parameter for the toCase() method is supposed to be false since PascalCasing is an on-demand feature. Therefore, I have kept the method as it is.